target/linux/realtek/files-5.10/drivers/clk/realtek/Kconfig

```
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: GPL-2.0-only
+config COMMON_CLK_RTL83XX
+	bool "Clock driver for Realtek RTL83XX"
+	depends on RTL83XX
```

```suggestion
	depends on COMMON_CLK_REALTEK && RTL83XX
```

and then one above 'enable support for realtek clocks to have this a sub-menu